packages/story-editor/src/components/storyFontPicker/index.js

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { forwardRef, useCallback } from '@googleforcreators/react';
+import PropTypes from 'prop-types';
+import { stripHTML } from '@googleforcreators/dom';
+
+/**
+ * Internal dependencies
+ */
+import { useFont, useStory } from '../../app';
+import useRichTextFormatting from '../panels/design/textStyle/useRichTextFormatting';
+import getClosestFontWeight from '../panels/design/textStyle/getClosestFontWeight';
+import { getCommonValue } from '../panels/shared';
+import FontPicker from '../fontPicker';
+import updateProperties from '../inspector/design/updateProperties';
+
+const StoryFontPicker = forwardRef(function StoryFontPicker({ ...rest }, ref) {
+  const { updateSelectedElements, selectedElements } = useStory(
+    ({ state, actions }) => ({
+      updateSelectedElements: actions.updateSelectedElements,
+      selectedElements: state.selectedElements,
+    })
+  );
+
+  const pushUpdate = useCallback(
+    (update, commitValues) => {
+      updateSelectedElements({
+        properties: (element) => {
+          const updates = updateProperties(element, update, commitValues);
+
+          return {
+            ...updates,
+          };
+        },
+      });
+    },
+    [updateSelectedElements]
+  );
```

Comment:
Locally I am seeing this get called twice, but in the unit test I am seeing it getting called 6 times. So I will look into why that might be. 

![Screen Shot 2022-03-08 at 2 41 30 PM](https://user-images.githubusercontent.com/1820266/157330101-ee30ab5a-0e85-4061-881e-db0b200a8a02.png)

https://github.com/GoogleForCreators/web-stories-wp/runs/5471047742?check_suite_focus=true#step:6:107